### PR TITLE
Some fix

### PR DESCRIPTION
**Pull request by P-Chang**
Fix of minor issue of voice and drawing.

Please check "hima_003.txt" and description of "dialog fix" strictly.

### CODE REVIEW

File: Update/hima_001.txt [hunk 1]
```diff
@@ -3136,7 +3136,6 @@ void main()
 	PlaySE(4, "s05/00/104300058", 128, 64);
 	OutputLine(NULL, "それを管理してるのは興宮市役所の人間。",
 		   NULL, " The person in charge of that is from the Okinomiya Municipal Office.", Line_WaitForInput);
-	PlaySE(4, "s05/00/104300059", 128, 64);
```
Comment (enumag) [inline Update/hima_001.txt]: Why did you remove this voice?

**Comment by enumag** [comment]
Thanks!

**Comment by ItaloKnox** [comment]
Any reason why you deleted the ``104300059`` voice? The text seems correct and unrelated to the previous voice.

**Comment by ItaloKnox** [comment]
Oh, actually I double checked it and the voice ``104300059`` covers a mix between 58 and 59. We should change the text, then.

**Comment by enumag** [comment]
I was wondering about that too at first but then I read the commit message and the 59 indeed seems to duplicate part of 58. If you can change the text to match better, that would be great.

**Comment by ItaloKnox** [comment]
I can't commit from this PC so if you can help me with that...

Japanese: ``それを管理してるのは地元の人間なんです。``
English: ``The person who's in charge of the Okinomiya Municipal Office is a local.``

**Comment by P-Chang** [comment]
If I translate voice.
" The person in charge of that is from a local. from a local"(in same tone)

It is really an ugly sound. 
I think that it is enough to just eliminate　"Line_WaitForInput"

**Comment by ItaloKnox** [comment]
Oops, I think I goofed in the english translation. Should be something like ``The person who's in charge of that is a local from the Okinomiya Municipal Office``.

Maybe that's better.

**Comment by enumag** [comment]
@ItaloKnox Would this work? I somehow dislike the `who's in charge of that` part.

`The person in charge is a local from the Okinomiya Municipal Office.`

**Comment by ItaloKnox** [comment]
Seems about right. I think it's already implicit that this person in particular is in charge of something.

**Comment by enumag** [comment]
Yep. I think it's clear from the context.

Can you chek the [commit](https://github.com/07th-mod/himatsubushi/commit/b7423e9e65f69396c01c0f6623afe6ed53d19b71) if I changed it the way you intended?

Btw. Maybe you're not aware but you can actually commit simple changes like this directly from the webbrowser, no need to clone the repo. I just used it for this change.

**Comment by ItaloKnox** [comment]
Yep, that's correct. 👍 

I actually knew you could do that but for some reason I always forget. Maybe I'm too used to change and commit directly from VS Code, so I rarely touch the code in the web interface.

**Comment by P-Chang** [comment]
Please leave the original text of Japanese, there is no big difference in sound and original have much information.

and I found reason. they cut it in PS3, PS2 have complete voice.
https://youtu.be/o3-2eZxA7s8?list=PLsL3bwCLzNutaX3GlmGmyI2xe6lQubNUL&t=1612
https://youtu.be/3CJVkizhJHg?list=PLff23ljXjapOa6mqAU5V0nhZpxH_cYTdo&t=1711

**Comment by ItaloKnox** [comment]
We should revert and use the PS2 voices, then. I actually think this is the better choice since it's closer to the original release.

**Comment by enumag** [comment]
@ItaloKnox https://github.com/07th-mod/himatsubushi/commit/020f1b353687c9fad66dee45b4fefb07b875b335
